lib/streamlit/elements/dataframe_selector.py

```diff
@@ -47,7 +47,7 @@ def dataframe(
 
         Parameters
         ----------
-        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table, Iterable, dict, or None
+        data : pandas.DataFrame, pandas.Styler, pyarrow.Table, numpy.ndarray, pyspark.sql.DataFrame, snowflake.snowpark.dataframe.DataFrame, snowflake.snowpark.table.Table, Iterable, dict, or None
```

Comment:
nit: for completeness, this, unfortunately, also needs to be added to the internal methods in `arrow` and vega lite.